python/vision_explanation_methods/error_labeling/example_notebook_error_labeling.ipynb

```diff
@@ -0,0 +1,272 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from responsibleai_vision import ModelTask, RAIVisionInsights\n",
```

Comment:
it looks like there are a lot of unused import here, like:

```
from responsibleai_vision import ModelTask, RAIVisionInsights
from responsibleai_vision.common.constants import ImageColumns
```

can you remove the unused ones here?